app/views/shared/components/_show_profiles.html.haml

```diff
@@ -1,6 +1,6 @@
 .small-12.medium-6.columns.text-right
   .profile-checkbox
-    %button.button.secondary.tiny.help-btn.ng-scope{:popover => t(:components_profiles_popover, sitename: Spree::Config[:site_name]), "popover-placement" => "left"}><
+    %button.button.secondary.tiny.right.help-btn.ng-scope{:popover => t(:components_profiles_popover, sitename: Spree::Config[:site_name]), "popover-placement" => "left"}><
```

Comment:
Sorry, I'm not sure what you mean. The other instances of `.tiny` should not be affected by this PR?

Reply:
Github is showing this diff for this commit:

![screenshot from 2018-05-30 14-36-26](https://user-images.githubusercontent.com/3524483/40699295-de321bec-6416-11e8-80f6-1e44b1e23bd4.png)

Doesn't that mean that `tiny` doesn't float right any more? In this line you add `right` to the button to make it float right. But all other elements of class `tiny` stop floating, right?

Reply:
Yeah, you’re right. What am I doing?!?!

Reply:
I've worked out what is happening with this. The `.tiny` selector you can see in that image is actually within a `.button.help-btn` selector.  So none of the other uses of `.tiny` (as documented in your original comment above) are affected by this change. So I think this can be marked as resolved?